django_project_base/js_app/src/definitions/projectList.js

```diff
@@ -30,12 +30,12 @@ const projectList = {
     },
     computed: {},
     methods: {
-      projectSelected(pk) {
-        if (pk === Store.get('current-project')) {
+      projectSelected(slug) {
+        if (slug === Store.get('current-project')) {
           return;
         }
-        Store.set('current-project', _.first(_.filter(this.projectList, p => p.id === pk)).id);
-        showNotification(null, 'project ' + pk + ' selected');
+        Store.set('current-project', _.first(_.filter(this.projectList, p => p.slug === slug)).slug);
```

Comment:
Ni mi všeč, da moraš v kodi popravljati id --> slug. Naredi property (getter, setter), ki bo to naredil zate na enem mestu, ti pa samo en property kličeš.

Reply:
Mislim, to si moral popraviti na kar nekaj mestih v kodi

Reply:
done